### PR TITLE
Najuma_Babirye_BCO_000556.json

### PR DESCRIPTION
**Pull request by nbabirye**
My BCO is ready for final review before pFDA submission.

### CODE REVIEW

File: Student_BCOs/Najuma_Babirye_BCO_000532.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+    "checksum": "ee2a0e2898aaf9e2955d45a2caec8dc5ee20aa46187a49a533d41c9370b13c3f",
     "bco_spec_version": "1.3.0",
     "provenance_domain": {
         "name": "RNA sequencing and gene enrichment of Pancreatic CTCs",
         "version": "1.0.0",
         "license": "https://spdx.org/licenses/CC-BY-4.0.html",
         "created": "2019-10-08T21:19:52.713920",
-        "modified": "2019-10-09T02:50:38.318710",
+        "modified": "2019-10-16T20:46:21.197743",
         "contributors": [
```
Comment (jpat1546) [inline Student_BCOs/Najuma_Babirye_BCO_000532.json:27]: @nbabirye not sure why I didn't flag this before, but all the publication authors need to be contributors. Sorry about not assigning this issue to you earlier. If you fix this, I'll review again. #113

**Review by jpat1546 — CHANGES_REQUESTED** [review]
@nbabirye 

I've reviewed your updated BCO- great job with modifying to include more information (extension domain, error domain)! 

I'm requesting you make one more change to this BCO- adding publication authors as contributors.

I should have noticed this when you first submitted, so I apologize for this late notice. I've assigned #113 to you- once you make this modification in the BCO Editor, download, and update this pull request with the new JSON. Once all changes have been made, assign this pull request back to me and I will review again. Until then, assigning this pull request back to you.

File: Student_BCOs/Najuma_Babirye_BCO_000532.json (hunk 1 above)
Comment (nbabirye) [reply Student_BCOs/Najuma_Babirye_BCO_000532.json:27]: @jpat1546,
I submitted a new commit with the changes for "authored by" added into the BCO. The BCO is ready for review again.
I just finished reassigning it to you.
thank you

Comment (jpat1546) [reply Student_BCOs/Najuma_Babirye_BCO_000532.json:27]: @nbabirye 2 more changes:

- In addition to "authoredBy", each publication author should also have the "derivedFrom" contribution type. Please add. See #113. 
- change review status of your BCO. See #148.

Once you make these changes in the BCO editor, please download, copy/paste JSON into this pull request. Thanks!

**Review by jpat1546 — CHANGES_REQUESTED** [review]
@nbabirye 2 more changes:

- In addition to "authoredBy", each publication author should also have the "derivedFrom" contribution type. Please add. See #113.
- change review status of your BCO. See #148.

Reassigning this pull request back to you. Once you make these changes in the BCO editor, please download, copy/paste JSON into this pull request. Thanks!

File: Student_BCOs/Najuma_Babirye_BCO_000532.json (hunk 1 above)
Comment (nbabirye) [reply Student_BCOs/Najuma_Babirye_BCO_000532.json:27]: @jpat1546 I'm still in Biochem lab,  (which started at 2pm) with limited access to a computer for this adjustment. If I can't get to it  before 5pm, I'll still commit the new change and let you know that I made it before submitting to precision FDA.

**Review by jpat1546 — APPROVED** [review]
### Your BCO is approved
Great job fixing all your assigned issues, @nbabirye (and in clutch time!) 

Prior to submitting to the pFDA challenge, please make the following 2 changes to your BCO in the BCO Editor (then download and submit to the pFDA):
In Provenance domain:

- Change reviewer status from "in review" to "approved"
- Change reviewer_comment from "Reviewed by BioCompute technical assistant, approval pending modification by author" to "Reviewed and approved by BioCompute technical assistant"

Leave me a comment if these instructions are not clear. Thanks!

**Pull request merged**